staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -508,7 +516,17 @@ func (c *Cacher) Watch(ctx context.Context, key string, opts storage.ListOptions
 	// given that memory allocation may trigger GC and block the thread.
 	// Also note that emptyFunc is a placeholder, until we will be able
 	// to compute watcher.forget function (which has to happen under lock).
-	watcher := newCacheWatcher(chanSize, filterWithAttrsFunction(key, pred), emptyFunc, c.versioner, deadline, pred.AllowWatchBookmarks, c.objectType, identifier)
+	watcher := newCacheWatcher(
+		chanSize,
+		filterWithAttrsFunction(key, pred),
+		emptyFunc,
+		c.versioner,
+		deadline,
+		pred.AllowWatchBookmarks,
+		c.objectType,
```

Comment:
The `objectType` field no longer needs to be in the `cacheWatcher` struct, and thus also no longer needs to be passed to `newCacheWatcher`.

Reply:
Updated, PTAL